index.mld

```diff
@@ -1,5 +1,5 @@
 {0 coq }
 
-The coq package is a virtual package gathering the coq-core and coq-stdlib packages.
+The coq package is a virtual package gathering the coq-core and rocq-core packages.
```

Comment:
This change is incorrect: `coq` still contains `coq-stdlib`. What would be correct would rather be this:
```suggestion
The coq package is a virtual package gathering the rocq-core and coq-stdlib packages.
```